pulp_python/app/models.py

```diff
@@ -72,6 +72,14 @@ class Meta:
             'filename',
         )
 
+    def __str__(self):
+        try:
+            # if we have a name, use it
+            return '<{}: {} ({})>'.format(self._meta.object_name, self.name, self.packagetype)
```

Comment:
This makes debugging nicer but it isn't necessary.  It's just a copy-paste from Content with the Python-specific "packagetype" added so you can tell whether the content is .whl, .tar.gz, etc.

We can delete it if it isn't desired.